DESCRIPTION

```diff
@@ -28,7 +28,8 @@ Imports:
     crayon,
     promises,
     sodium,
-    swagger (> 3.20.3)
+    swagger (> 3.20.3),
+    readr
```

Comment:
Let's make `readr` a suggested package, please.  It is not needed and is useful for only a part of the code.

Reply:
I might have to undo the change/request after merging if deploying the API doesn't catch that `readr` is needed.